### PR TITLE
Use latest xmosdoc and xjsl versions

### PR DESCRIPTION
**Pull request by lucianomartin**
Part of https://xmosjira.atlassian.net/browse/XVC-79.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -9,9 +9,6 @@ XCORE |reg| -VOICE Solutions
 
   doc/quick_start_guide/index
   doc/programming_guide/index
-  modules/voice/doc/user_guide/audio_processing/index
```
Comment (mbanth) [inline index.rst:12]: Please reverse this change and the two immediately below.

**Review by mbanth — CHANGES_REQUESTED** [review]

File: doc/exclude_patterns.inc [hunk 2]
```diff
@@ -4,12 +4,12 @@ modules/core/modules/legacy_compat
...
 modules/core/modules/random
 modules/core/modules/trycatch
-modules/rtos/modules/FreeRTOS
+modules/rtos/**
 modules/inferencing
 modules/io/modules/xud
 modules/io/doc/substitutions.rst
 modules/sample_rate_conversion
-modules/voice/doc/substitutions.rst
+modules/voice/**
```
[the remark this answers is not in the document]
Comment (lucianomartin) [reply doc/exclude_patterns.inc]: Done in https://github.com/xmos/sln_voice/pull/367/commits/c64b2edb12515936abc8ebe0196fd3db07a2d64a.

File: doc/exclude_patterns.inc [hunk 3]
```diff
@@ -4,12 +4,12 @@ modules/core/modules/legacy_compat
 modules/core/modules/otpinfo
 modules/core/modules/random
 modules/core/modules/trycatch
-modules/rtos/modules/FreeRTOS
+modules/rtos/**
```
[the remark this answers is not in the document]
Comment (lucianomartin) [reply doc/exclude_patterns.inc]: Done in https://github.com/xmos/sln_voice/pull/367/commits/c64b2edb12515936abc8ebe0196fd3db07a2d64a.

File: index.rst (hunk 1 above)
Comment (lucianomartin) [reply index.rst]: Done in https://github.com/xmos/sln_voice/pull/367/commits/c64b2edb12515936abc8ebe0196fd3db07a2d64a.

**Review by mbanth — APPROVED** [review]

**Pull request merged**